File: DesafioTecnicoFrame_Java_2020/src/View/TODOsView.java
package View;

import Controller.TODOsController;
import Model.TODOs;

public class TODOsView {

    /*
     * Metodo responsável por imprimir a requisição
     * de To DOs.
     */
    public void showPost( ){
        TODOsController tc = new TODOsController();

        for (TODOs todos : tc.getTODOs()){
            System.out.println(todos);
        }//end for
    }//end showPost();

}//end class

File: DesafioTecnicoFrame_C#_2020/DesafioTecnico/View/TODOsView.cs
using DesafioTecnico.Controller;
using System;
using System.Threading.Tasks;

namespace DesafioTecnico.View
{
    public class TODOsView
    {
        //Imprimir requisições
        public void showPost()
        {
            Console.WriteLine("\nConsultando...\n");
            var todos = new TODOsController();
            var todosTask = todos.GetTODOsAsync();

            todosTask.ContinueWith(task =>
            {
                var usuarios = task.Result;
                foreach (var u in usuarios)
                    Console.WriteLine(u.ToString() + "\n");
                Console.Write("Pressione ENTER para retornar ao menu!");
            },
             TaskContinuationOptions.RunContinuationsAsynchronously

             );

            Console.ReadLine();

        }//end showPost
    }
}
